crates/sui-types/src/move_package.rs

```diff
@@ -32,6 +32,8 @@ use std::collections::BTreeMap;
 
 pub const PACKAGE_MODULE_NAME: &IdentStr = ident_str!("package");
 pub const UPGRADECAP_STRUCT_NAME: &IdentStr = ident_str!("UpgradeCap");
+pub const UPGRADETICKET_STRUCT_NAME: &IdentStr = ident_str!("UpgradeTicket");
```

Comment:
I think the changes to this file could technically go into your second PR, but not a big deal.